WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -31,6 +31,8 @@ namespace WalletWasabi.Gui.ViewModels
 {
 	public class StatusBarViewModel : ViewModelBase
 	{
+		private const string LoadingText = "Loading ...";
```

Comment:
Can we change this to "Updating ..."

Reply:
But when Wasabi is connecting, the price is not updating. The price is retrieved / loading / unknown ...

Reply:
I would say that "Updating ..." is not exactly the correct word.

Also there is already "Loading..." in the file: https://github.com/zkSNACKs/WalletWasabi/blob/23958b6f92f7f8a9622d06269531c2b9d332fd1b/WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs#L150

Reply:
Ok!